components/eamxx/src/share/property_checks/mass_and_energy_column_conservation_check.hpp

```diff
@@ -26,10 +29,14 @@ class MassAndEnergyColumnConservationCheck: public PropertyCheck {
   template <typename S>
   using uview_2d = typename ekat::template Unmanaged<view_2d<S> >;
 
+  //using namespace ShortFieldTagsNames;
+  //using namespace ekat::units;
+
 public:
 
   // Constructor
-  MassAndEnergyColumnConservationCheck (const std::shared_ptr<const AbstractGrid>& grid,
+  MassAndEnergyColumnConservationCheck (const ekat::Comm& comm,
+                                        const std::shared_ptr<const AbstractGrid>& grid,
```

Comment:
FYI, the grid already stores a comm object, so you can also get it by `grid->get_comm()`.

Reply:
thanks, will do it in the next PR.